src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
+import Navbar from './components/Navbar';
+import MyProfile from './components/My-Profile';
+import Dragons from './components/Dragons';
+import MissionList from './components/Missions/MissionList';
+import Rockets from './components/rockets/rockets';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Navbar />
```

Comment:
- [ ] please align the images for the rockets and dragons. so they all can be in order
 - [ optional] you can also make the reserved text to show red so a user takes it serious and easily notice that is has been reserved 
![Screenshot (413)_LI](https://user-images.githubusercontent.com/65298048/188245533-cd726761-710d-46c4-864e-78f06ee51d2b.jpg)
![Screenshot (414)_LI](https://user-images.githubusercontent.com/65298048/188245555-55e641ce-8df6-4982-9ba6-e44aea7b1626.jpg)

good example
![Screenshot (415)](https://user-images.githubusercontent.com/65298048/188245646-c8be8c5a-698d-4469-90c5-35723d727fab.png)